src/lib.rs

```diff
@@ -181,12 +181,12 @@ pub fn derive_error(input: TokenStream) -> TokenStream {
         format!(
             r#"impl ::{std_crate}::fmt::Display for {name} {{
                 fn fmt(&self, f: &mut ::{std_crate}::fmt::Formatter<'_>) ->
-                    Result<(), ::{std_crate}::fmt::Error>
+                    ::std::result::Result<(), ::{std_crate}::fmt::Error>
```

Comment:
This sounds like a good improvement, but it really needs to be `::{std_crate}::result::Result` so that it remains compatible with `#[no_std]`. With that change you'll have my 👍

Reply:
Sure, probably should have noticed that from the other usages.